src/OrchardCore.Modules/OrchardCore.Users/Startup.cs

```diff
@@ -147,7 +148,8 @@ public override void ConfigureServices(IServiceCollection services)
                 // Note: it has no practical impact on the security of the site since all endpoints are always
                 // protected by antiforgery checks, that are enforced with or without this setting being changed.
                 // 2019-12-10; Removed, since https://github.com/aspnet/Announcements/issues/390
-                // options.Cookie.SameSite = SameSiteMode.None;
+                // 2020-02-17; Reenabled since we have compensation logic for backwardscompatibility
+                options.Cookie.SameSite = SameSiteMode.None;
             });
```

Comment:
Isn't this always overridden when you use the Cookie Policy middleware?

Reply:
Was meaning to add the NotReady label, since I want feedback like this in order to not miss anything crucial.

Reply:
And to your comment;
I don't think so.
Since we hook into the events when creating a cookie, we change it at the last possible time.
So this is the default setting and we adjust it when we detect a useragent that is not compatible with this setting.